packages/react/src/Button/Button.features.stories.tsx

```diff
@@ -16,7 +16,7 @@ export const Primary = () => (
 )
 
 export const PrimaryDisabled = () => (
-  <Button as="a" disabled variant="primary" href="#">
+  <Button as="button" disabled variant="primary" href="#">
```

Comment:
Can we remove the `href` in this example too if we're using `button`?

```suggestion
  <Button as="button" disabled variant="primary">
```

Reply:
Definitely, good call! I removed the `href` if the component was either the default (button) or `as="button"`.